### ISSUE TITLE
Return JSON results (for classes service) following the Breeding API specification

### ISSUE THREAD

**Issue by jonquet** [agroportal/project-management#25]
The Breeding API specifies how to return Traits & Observation Variables
http://docs.brapi.apiary.io/#

For a given ontology, we can have the /classes service
http://data.agroportal.lirmm.fr/documentation#Class
that returns classes aggregated in a certain way that (variable=trait+method+scale) to be compliant with the Breedding API

We can do that in our Tomcat/Java servlet while post-processing the result of /classes service
http://services.agroportal.lirmm.fr/

Basically, it means deploying a new service:
http://services.agroportal.lirmm.fr/brapi/v1/traits
http://services.agroportal.lirmm.fr/brapi/v1/traits/id
http://services.agroportal.lirmm.fr/brapi/v1/variables/datatypes
http://services.agroportal.lirmm.fr/brapi/v1/variables
http://services.agroportal.lirmm.fr/brapi/v1/variables/id

See spec on Breeding API web site.

**Comment by syphax-bouazzouni** [comment]
is this still relevant @jonquet?

**Comment by jonquet** [comment]
Yes. Just waiting for a driving use case with @cpommier or someone else ;)

**Comment by jonquet** [comment]
We need to see the exact output for this (with  @cpommier) then it will be implemented in the proxy. 
See also : https://cropontology.org/api_help

**Comment by jonquet** [comment]
The idea would be to builf dor instance out of this service call: 
https://agroportal.lirmm.fr/ontologies/CO_321/?p=classes&conceptid=https%3A%2F%2Fcropontology.org%2Frdf%2FTrait

All traits behind subclasses of : https://cropontology.org/rdf/Trait
Or another another set of classes to be specified.

**Comment by cpommier** [comment]
Hi, I updated CO_321 in https://github.com/Planteome/CO_321-wheat-traits
